### ISSUE TITLE
A speaker image appears near the RGFW application's icon when an audio source is played

### ISSUE THREAD

**Issue by EimaMei** [ColleagueRiley/RGFW#20]
As the title suggests, anytime music is being played from an application, a speaker icon appears near the RGFW application's. I presume this is a DE "feature" so to speak that has to be enabled/disabled via some WM-specific X11 atom. 

**Example 1: Audio is being played from the Audacious app**
![image](https://github.com/ColleagueRiley/RGFW/assets/69605234/7dedf5ae-18ed-40af-9a57-1e6bd7294ab9)
**Example 2: No audio is being played at all**
![image](https://github.com/ColleagueRiley/RGFW/assets/69605234/1b4cca66-0fae-4adb-ab7c-fff3f43814ef)

Note that in the first example image, the speaker icon doesn't appear for other applications that have audio-playing capabilities, like Chromium. This makes me believe that GTK has solved this issue somewhere in their code base, so the solution most likely lies somewhere in their source.

If this helps, here is additional information about the window manager (Kwin) that has this issue:
```
==========================

Version
=======
KWin version: 6.1.1
Qt Version: 6.7.2
Qt compile version: 6.7.2
XCB compile version: 1.17.0

Operation Mode: X11 only

Build Options
=============
KWIN_BUILD_DECORATIONS: yes
KWIN_BUILD_TABBOX: yes
KWIN_BUILD_ACTIVITIES: yes
HAVE_X11_XCB: yes
HAVE_GLX: yes

X11
===
Vendor: The X.Org Foundation
Vendor Release: 12101013
Protocol Version/Revision: 11/0
SHAPE: yes; Version: 0x11
RANDR: yes; Version: 0x14
DAMAGE: yes; Version: 0x11
Composite: yes; Version: 0x4
RENDER: yes; Version: 0xb
XFIXES: yes; Version: 0x50
SYNC: yes; Version: 0x31
GLX: yes; Version: 0x0

Decoration
==========
Plugin: org.kde.breeze
Theme: 
Plugin recommends border size: None
onAllDesktopsAvailable: false
alphaChannelSupported: true
closeOnDoubleClickOnMenu: false
decorationButtonsLeft: 0, 2
decorationButtonsRight: 6, 3, 4, 5
borderSize: 0
gridUnit: 18
font: Noto Sans,10,-1,0,400,0,0,0,0,0,0,0,0,0,0,1
smallSpacing: 4
largeSpacing: 18

Output backend
==============
Name: KWin::X11StandaloneBackend

Cursor
======
themeName: breeze_cursors
themeSize: 36

Options
=======
focusPolicy: ClickToFocus
xwaylandCrashPolicy: 1
xwaylandMaxCrashCount: 3
nextFocusPrefersMouse: false
clickRaise: true
autoRaise: false
autoRaiseInterval: 0
delayFocusInterval: 0
shadeHover: false
shadeHoverInterval: 250
separateScreenFocus: false
placement: 5
activationDesktopPolicy: SwitchToOtherDesktop
focusPolicyIsReasonable: true
borderSnapZone: 10
windowSnapZone: 10
centerSnapZone: 0
snapOnlyWhenOverlapping: false
edgeBarrier: 100
cornerBarrier: 1
rollOverDesktops: false
focusStealingPreventionLevel: 1
operationTitlebarDblClick: 5000
operationMaxButtonLeftClick: 5000
operationMaxButtonMiddleClick: 5015
operationMaxButtonRightClick: 5014
commandActiveTitlebar1: MouseRaise
commandActiveTitlebar2: MouseNothing
commandActiveTitlebar3: MouseOperationsMenu
commandInactiveTitlebar1: MouseActivateAndRaise
commandInactiveTitlebar2: MouseNothing
commandInactiveTitlebar3: MouseOperationsMenu
commandWindow1: MouseActivateRaiseAndPassClick
commandWindow2: MouseActivateAndPassClick
commandWindow3: MouseActivateAndPassClick
commandWindowWheel: MouseNothing
commandAll1: MouseUnrestrictedMove
commandAll2: MouseToggleRaiseAndLower
commandAll3: MouseUnrestrictedResize
keyCmdAllModKey: 16777250
condensedTitle: false
electricBorderMaximize: true
electricBorderTiling: true
electricBorderCornerRatio: 0.25
borderlessMaximizedWindows: false
killPingTimeout: 5000
hideUtilityWindowsForInactive: true
compositingMode: 1
useCompositing: true
hiddenPreviews: 1
glSmoothScale: 2
glStrictBinding: true
glStrictBindingFollowsDriver: true
glPreferBufferSwap: AutoSwapStrategy
glPlatformInterface: 1
windowsBlockCompositing: true
allowTearing: true

Screen Edges
============
desktopSwitching: false
desktopSwitchingMovingClients: false
cursorPushBackDistance: 1x1
timeThreshold: 75
reActivateThreshold: 350
actionTopLeft: 0
actionTop: 0
actionTopRight: 0
actionRight: 0
actionBottomRight: 0
actionBottom: 0
actionBottomLeft: 0
actionLeft: 0

Screens
=======
Number of Screens: 1

Screen 0:
---------
Name: eDP-1
Enabled: 1
Geometry: 0,0,3840x2160
Scale: 1
Refresh Rate: 59996
Adaptive Sync: incapable

Compositing
===========
Compositing is active
Compositing Type: OpenGL
OpenGL vendor string: AMD
OpenGL renderer string: AMD Radeon RX Graphics (radeonsi, polaris11, LLVM 17.0.6, DRM 3.57, 6.9.6-arch1-1)
OpenGL version string: 4.6 (Compatibility Profile) Mesa 24.1.2-arch1.1
OpenGL platform interface: GLX
OpenGL shading language version string: 4.60
Driver: Unknown
GPU class: Unknown
OpenGL version: 4.6
GLSL version: 4.60
Mesa version: 24.1.2
X server version: 1.21.1
Linux kernel version: 6.9.6
Direct rendering: Requires strict binding: yes
Virtual Machine:  no
OpenGL 2 Shaders are used

Loaded Effects:
---------------
screenshot
outputlocator
colorpicker
zoom
screenedge
blur
contrast
sessionquit
logout
login
slidingpopups
windowaperture
slide
squash
scale
morphingpopups
maximize
fullscreen
frozenapp
fadingpopups
dialogparent
windowview
tileseditor
overview
highlightwindow
blendchanges
startupfeedback
kscreen

Currently Active Effects:
-------------------------
blur
contrast

Effect Settings:
----------------
screenshot:

outputlocator:

colorpicker:

zoom:
zoomFactor: 1.2
mousePointer: 0
mouseTracking: 0
focusTrackingEnabled: false
textCaretTrackingEnabled: false
focusDelay: 350
moveFactor: 20
targetZoom: 1

screenedge:

blur:

contrast:

sessionquit:
pluginId: sessionquit
isActiveFullScreenEffect: false

logout:
pluginId: logout
isActiveFullScreenEffect: false

login:
pluginId: login
isActiveFullScreenEffect: false

slidingpopups:
slideInDuration: 200
slideOutDuration: 200

windowaperture:
pluginId: windowaperture
isActiveFullScreenEffect: false

slide:
horizontalGap: 45
verticalGap: 20
slideBackground: true

squash:
pluginId: squash
isActiveFullScreenEffect: false

scale:
pluginId: scale
isActiveFullScreenEffect: false

morphingpopups:
pluginId: morphingpopups
isActiveFullScreenEffect: false

maximize:
pluginId: maximize
isActiveFullScreenEffect: false

fullscreen:
pluginId: fullscreen
isActiveFullScreenEffect: false

frozenapp:
pluginId: frozenapp
isActiveFullScreenEffect: false

fadingpopups:
pluginId: fadingpopups
isActiveFullScreenEffect: false

dialogparent:
pluginId: dialogparent
isActiveFullScreenEffect: false

windowview:
activeView: 
delegate: 
animationDuration: 300
ignoreMinimized: false
mode: 
partialActivationFactor: 0
gestureInProgress: false
searchText: 
selectedIds: 

tileseditor:
activeView: 
delegate: 
animationDuration: 200

overview:
activeView: 
delegate: 
animationDuration: 300
ignoreMinimized: false
filterWindows: true
organizedGrid: true
overviewPartialActivationFactor: 0
overviewGestureInProgress: false
transitionPartialActivationFactor: 0
transitionGestureInProgress: false
gridPartialActivationFactor: 0
gridGestureInProgress: false
desktopOffset: 
searchText: 

highlightwindow:

blendchanges:

startupfeedback:
type: 1

kscreen:


Loaded Plugins:
---------------
krunnerintegration
nightlight

Available Plugins:
------------------
BounceKeysPlugin
StickyKeysPlugin
buttonsrebind
eis
krunnerintegration
nightlight
screencast
```

**Comment by ColleagueRiley** [comment]
I don't understand this issue. Do you mean this behavior only happens when RGFW is open?

**Comment by EimaMei** [comment]
The icon appears for both the application that is playing the audio and the RGFW application, even though no audio is being played through the RGFW app

**Comment by ColleagueRiley** [comment]
Ok, thank you for clarifying.

**Comment by ColleagueRiley** [comment]
I'm unable to recreate this issue with KDE (plasma / X11)

**Comment by ColleagueRiley** [comment]
![Screenshot_2024-07-02_20-20-53](https://github.com/ColleagueRiley/RGFW/assets/50218871/8c61a6b3-cf38-4475-8cb0-b97bc0e9d0f7)

**Comment by EimaMei** [comment]
UPDATE: I might've found the "culprit". Running a basic X11 application (like `vkcube`) still has the same speaker issue, while RGFW is fine.
![image](https://github.com/ColleagueRiley/RGFW/assets/69605234/89c4edcb-cb80-4c25-81b2-70a25cf0c6a2)

This made me incredibly suspicious and wonder if the recently merged PR (#19) had to do with any of this. Turns out, removing that scope and recompiling the example does make the issue resurface.

![image](https://github.com/ColleagueRiley/RGFW/assets/69605234/49055353-b920-440c-912c-2ebfbef0e3e7)

Not sure why KDE adds the mute toggle for non-class hinted applications, but at least this discovery confirms that the issue WAS presented and is fixed for KDE users
